packages/lib/src/allTimeZones.ts

```diff
@@ -0,0 +1,110 @@
+const allTimeZones = [
```

Comment:
Can you rename it with `label` and `value`? I'd find it cleaner. And we won't even have to re-map it